LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2013 Yamashita, Yuu
+Copyright (c) 2013-2019 Yamashita, Yuu
```

Comment:
Let's add your name as well 😄

Reply:
Thank you, but as I am removing what I contributed, it makes no sense to me anymore to be mentioned in the copyright :)